src/middleware/share.ts

```diff
@@ -0,0 +1,164 @@
+import { Request, Response, NextFunction } from 'express'
+// import { server } from 'decentraland-server'
+import { env } from 'decentraland-commons'
+
+const BUILDER_DOMAIN = env.get('BUILDER_DOMAIN')
+if (!BUILDER_DOMAIN) {
```

Comment:
I think this should be `BUILDER_URL`? or maybe it's missing from `.env.example`?